core/src/main/java/io/confluent/kafka/schemaregistry/metrics/MetricsContainer.java

```diff
@@ -201,16 +201,6 @@ private SchemaRegistryMetric getSchemaTypeMetric(String type, boolean isRegister
     }
   }
 
-  private static List<String> getMetricReporterConfig(SchemaRegistryConfig config) {
-    List<String> classes = new ArrayList<>(config.getList(
-            ProducerConfig.METRIC_REPORTER_CLASSES_CONFIG));
-    if (config.getBoolean(SchemaRegistryConfig.TELEMETRY_REPORTER_ENABLED_CONFIG)
```

Comment:
Are we not using TELEMETRY_REPORTER_ENABLED_CONFIG any longer?  Should we remove it?

Reply:
Good catch, fixed !